ontology/vg.ttl

```diff
@@ -113,6 +114,60 @@
   rdfs:label "reverseOfNode"^^xsd:string ;
   rdfs:range :Node ;
 .
+:Bin
+  rdf:type owl:Class ;
+  rdfs:comment "A bin in the output of odgi bin, representing a sequence section."^^xsd:string ;
+  rdfs:label "Bin"^^xsd:string ;
+  rdfs:subClassOf owl:Thing ;
+.
+:Link
+  rdf:type owl:Class ;
+  rdfs:comment "An arrow marks a graph traversal along a nonlinear connection."^^xsd:string ;
+  rdfs:label "Arrow"^^xsd:string ;
+  rdfs:subClassOf owl:Thing ;
+.
+:binRank
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "The super property that says two bins are linked on the PangenomeSequence."^^xsd:string ;
+  rdfs:domain :Bin ;
+  rdfs:label "connects"^^xsd:string ;
+  rdfs:range :Bin ;
+.
+:binEdge
+  rdf:type owl:DatatypeProperty ;
+  rdfs:comment "The rank records that step place along its panngenome sequence."^^xsd:string ;
+  rdfs:domain :Bin ;
+  rdfs:label "rank"^^xsd:string ;
+  rdfs:range xsd:positiveInteger ;
+.
+:binRegion
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "A bin correspnds to the specific region of bins."^^xsd:string ;
+  rdfs:domain :Bin ;
+  rdfs:label "is"^^xsd:string ;
+  rdfs:range faldo:Region ;
+.
+:arrival
```

Comment:
It looks like `:arrival` is the `owl:inverseOf` of `:departure`.

Reply:
That's true. I will add the line.